mne/viz/epochs.py

```diff
@@ -447,3 +448,123 @@ def plot_epochs(epochs, epoch_idx=None, picks=None, scalings=None,
+                             'must also be supplied')
+        megs = ['mag', 'grad', False]
+        eegs = [False, False, True]
+        names = ['Magnetometers', 'Gradiometers', 'EEG']
+        picks_list = list()
+        titles_list = list()
+        for meg, eeg, name in zip(megs, eegs, names):
+            picks = pick_types(epochs.info, meg=meg, eeg=eeg, ref_meg=False)
+            if len(picks) > 0:
+                picks_list.append(picks)
+                titles_list.append(name)
+        if len(picks_list) == 0:
+            raise RuntimeError('No MEG or EEG channels found')
+    else:
+        picks_list = [picks]
+        titles_list = ['Selected channels']
+        ax_list = [ax]
+
+    make_label = False
+    fig = None
+    if ax is None:
+        fig = plt.figure()
+        ax_list = list()
+        for ii in range(len(picks_list)):
+            # Make x-axes change together
+            if ii > 0:
+                ax_list.append(plt.subplot(len(picks_list), 1, ii + 1,
+                                           sharex=ax_list[0]))
+            else:
+                ax_list.append(plt.subplot(len(picks_list), 1, ii + 1))
+        make_label = True
+    else:
+        fig = ax_list[0].get_figure()
+
+    for ii, (picks, title, ax) in enumerate(zip(picks_list, titles_list,
+                                                ax_list)):
+        psds, freqs = compute_epochs_psd(epochs, picks=picks, fmin=fmin,
+                                         fmax=fmax, n_fft=n_fft,
+                                         window_size=window_size,
+                                         n_overlap=n_overlap, proj=proj,
+                                         n_jobs=n_jobs)
+
+        # Convert PSDs to dB
+        psds = 10 * np.log10(psds)
+        # mean across epochs and channels
+        psd_mean = np.mean(psds, axis=0).mean(axis=0)
+        if area_mode == 'std':
+            # std across channels
+            psd_std = np.std(np.mean(psds, axis=0), axis=0)
+            hyp_limits = (psd_mean - psd_std, psd_mean + psd_std)
+        elif area_mode == 'range':
+            hyp_limits = (np.min(np.mean(psds, axis=0), axis=0),
+                          np.max(np.mean(psds, axis=0), axis=0))
+        else:  # area_mode is None
+            hyp_limits = None
+
+        ax.plot(freqs, psd_mean, color=color)
+        if hyp_limits is not None:
+            ax.fill_between(freqs, hyp_limits[0], y2=hyp_limits[1],
+                            color=color, alpha=area_alpha)
+        if make_label:
+            if ii == len(picks_list) - 1:
+                ax.set_xlabel('Freq (Hz)')
+            if ii == len(picks_list) / 2:
+                ax.set_ylabel('Power Spectral Density (dB/Hz)')
+            ax.set_title(title)
+            ax.set_xlim(freqs[0], freqs[-1])
+    if make_label:
+        tight_layout(pad=0.1, h_pad=0.1, w_pad=0.1, fig=fig)
+    plt.show()
+    return fig
```

Comment:
Maybe now or later we should have method raw/epochs.plot_psds_topomap to get a figure like here:

http://martinos.org/mne/stable/auto_examples/time_frequency/plot_single_trial_spectra.html

Although with a better vmin / vmax handling, maybe normalization (dividing each band by sum over frequencies).